hexagon/support/storage.py

```diff
@@ -0,0 +1,191 @@
+import os
+from pathlib import Path
+import sys
+from typing import Any, Dict, List
+from ruamel.yaml import YAML
+from enum import Enum
+from shutil import rmtree
+from hexagon.cli import cli, configuration
+
+HEXAGON_STORAGE_APP = "hexagon"
+
+
+class HexagonStorageKeys(Enum):
+    last_command = "last-command"
+
+
+class StorageValueType(Enum):
+    text = "text"
+    text_multiline = "text-multilne"
+    dictionary = "dictionary"
+
+
+_extension_by_value_type = {
+    StorageValueType.text: ".txt",
+    StorageValueType.text_multiline: ".txt",
+    StorageValueType.dictionary: ".yaml",
+}
+
+_storage_path_by_os = {
+    "linux": os.path.expanduser("~/.config/hexagon"),
+    "darwin": os.path.expanduser("~/.config/hexagon"),
+    "cygwin": os.path.expanduser("~/.config/hexagon"),
+    "win32": os.path.expanduser("~/hexagon"),
+}
+
+_storage_dir_path = None
+
+InputDataType = str or List[str] or Dict[Any]
+
+
+def _merge_dictionaries_deep(a, b, path=None):
+    "merges b into a"
```

Comment:
este docstring debería tener 3 comillas
```suggestion
    """
    merges b into a
    """
```